### ISSUE TITLE
Document content/headers of data download files

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3153]
**Document content/headers of data download files**

We should document the columns included in the SIMPLE, Darwin Core Archive and Species downloads.
For a start we could make an FAQ explaining that most of our columns correspond to Darwin Core terms and that their definitions can be found here: https://dwc.tdwg.org/terms/

-----

Github user: @ManonGros
User: [See in registry](https://www.gbif.org/api/feedback/user/bce71ac839569962cf06c510af3b921b:22da2afd6f76503b32ae378f2275bb829db819efeca9f7bd1341d2c6b4b6badc88319b06bed755303a4de5a4dd965da8fe3de065bc6dd3a7eaa9797a5d0219fa)
System: Safari 14.0.0 / Mac OS X 10.15.7
Referer: https://www.gbif.org/faq?q=download
Window size: width 1608 - height 832
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-12-14T15:55:17.628Z',mode:absolute,to:'2020-12-14T16:01:17.628Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-12-14T15:55:17.628Z',mode:absolute,to:'2020-12-14T16:01:17.628Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
Something like this! https://github.com/microsoft/AIforEarthDataSets/blob/main/data/gbif.md
For the columns in DwC downloads, we can do something with minimum documentation like:
_column, data type, required, see dwc:term_

**Comment by JimBacon** [comment]
Yes please! That documentation is exactly what I have been searching for.
In particular, can you document what is deduced in https://github.com/gbif/portal-feedback/issues/1201, that the parent_key of synonyms points to the accepted name.

**Comment by JimBacon** [comment]
I'm also stumbling over which fields I can expect to see populated. For example, every record has a scientific_name but I have been caught out because many do not have a canonical_name. Perhaps this could be documented too.

An example such as id=10863734 (scientific_name Mareca americana x penelope), which is an accepted name, lacks canonical_name, genus_or_above, specific_epithet, infra_specific_epithet, and notho_type though I would have expected them. I'm unclear if this is malformed data which has slipped through quality control or valid data that I have to be able to handle.

**Comment by CecSve** [comment]
This issue is relevant for the documentation work as well https://github.com/gbif/portal-feedback/issues/4020
